docs/content/doc/installation/from-package.en-us.md

```diff
@@ -47,9 +47,52 @@ pacman -S gitea
 
 There is a [Gitea Snap](https://snapcraft.io/gitea) package which follows the latest stable version.
 
-``sh
-snap install gitea
-``
+### This guide was tested with Ubuntu 20.04 and MySQL:
+
+* Install needed Packages and the snap:
+
+```
+sudo snap install gitea
+sudo apt install nginx mariadb-server
+```
+* Secure your mysql Installation
```

Comment:
```suggestion
* Secure your MySQL installation:
```

Reply:
It seems that this document is about MariaDB, which is slightly different from MySQL, especially in the database authentication part. And I do not think this document should contain too many things about database installation.

Reply:
I tried to resolve these issues with the following commit:
https://github.com/go-gitea/gitea/pull/17040/commits/5f3fdcae8c25abe94cf69645e4407e21070aa9de

Reply:
Tried resolving this issues with this commit: https://github.com/go-gitea/gitea/pull/17040/commits/5f3fdcae8c25abe94cf69645e4407e21070aa9de